src/test/java/org/sagebionetworks/web/unitclient/widget/entity/controller/EntityActionControllerImplTest.java

```diff
@@ -703,16 +703,15 @@ public void testEditFileMetadataIsCurrent(){
 	
 	@Test
 	public void testEditFileMetadataIsNotCurrent(){
-		Entity file = new FileEntity();
+		FileEntity file = new FileEntity();
 		file.setId(entityId);
 		file.setParentId(parentId);
+		file.setVersionNumber(1L);
 		entityBundle.setEntity(file);
 		// currentPlace returns a non-null versionNumber
-		Synapse currentPlace = new Synapse(currentUserId, 1L, null, currentUserId);
-		when(mockGlobalApplicationState.getCurrentPlace()).thenReturn(currentPlace);
 		AsyncMockStubber.callWithInvoke().when(mockPreflightController).checkUpdateEntity(any(EntityBundle.class), any(Callback.class));
 		AsyncMockStubber.callNoInvovke().when(mockEditFileMetadataModalWidget).configure(any(FileEntity.class), anyString(), any(Callback.class));
-		controller.configure(mockActionMenu, entityBundle, wikiPageId,mockEntityUpdatedHandler);
+		controller.configure(mockActionMenu, entityBundle, false, wikiPageId, mockEntityUpdatedHandler);

```

Comment:
False is tested, just mostly is irrelevant because non-versionables or unrelated components are tested